core/vm/runtime/runtime.go

```diff
@@ -113,17 +113,14 @@ func Execute(code, input []byte, cfg *Config) ([]byte, *state.StateDB, error) {
 		vmenv   = NewEnv(cfg)
 		sender  = vm.AccountRef(cfg.Origin)
 	)
-	// TODO(meowsbits): re: EIP2929
-	/*
-		if cfg.ChainConfig.IsYoloV2(vmenv.BlockNumber) {
-			cfg.State.AddAddressToAccessList(cfg.Origin)
-			cfg.State.AddAddressToAccessList(address)
-			for _, addr := range vmenv.ActivePrecompiles() {
-				cfg.State.AddAddressToAccessList(addr)
-				cfg.State.AddAddressToAccessList(addr)
-			}
+	if cfg.ChainConfig.IsEnabled(cfg.ChainConfig.GetEIP2929Transition, vmenv.BlockNumber) {
+		cfg.State.AddAddressToAccessList(cfg.Origin)
+		cfg.State.AddAddressToAccessList(address)
+		for addr := range vm.PrecompiledContractsForConfig(cfg.ChainConfig, vmenv.BlockNumber) {
+			cfg.State.AddAddressToAccessList(addr)
```

Comment:
The duplicated line here, is already being tracked at [go-ethereum](https://github.com/ethereum/go-ethereum/pull/21956)